macros.cfg

```diff
@@ -117,6 +109,28 @@ gcode:
   G1 Y100 F{speed}
   # Reset extrusion
   G92 E0
+  
+[gcode_macro PARK]
+gcode:
+  {% set speed = printer["gcode_macro RatOS"].macro_travel_speed|float * 60 %}
+  # Get X position
+  {% if params.X is defined and params.X|float >= printer.toolhead.axis_minimum.x + 5 and params.X|float <= printer.toolhead.axis_maximum.x - 5 %}
+    {% set safe_x = params.X|float %}
+  {% else %}
+    {% set safe_x = printer.toolhead.axis_maximum.x / 2 %}
```

Comment:
Probably issue a warning here to notify the user if his requested X coordinate is out of bounds

Reply:
`{action_respond_warning(msg)}` that is

Reply:
OK let me know what you think to this setup